amundsen_application/static/js/components/common/Breadcrumb/index.tsx

```diff
@@ -1,14 +1,28 @@
 import * as React from 'react';
+import { connect } from 'react-redux';
 import { Link } from 'react-router-dom';
 
 import './styles.scss';
+import { GlobalState } from 'ducks/rootReducer';
 
-export interface BreadcrumbProps {
+export interface OwnProps {
   path: string;
   text: string;
 }
 
-const Breadcrumb: React.SFC<BreadcrumbProps> = ({ path, text }) => {
+export interface StateFromProps {
+  searchTerm: string;
+}
+
+export type BreadcrumbProps = OwnProps & StateFromProps;
+
+export const Breadcrumb: React.SFC<BreadcrumbProps> = (props) => {
+  let path = props.path;
+  let text = props.text;
+  if (props.searchTerm) {
+    path = `/search?searchTerm=${props.searchTerm}&selectedTab=table&pageIndex=0`
```

Comment:
Also worth considering: You could do `path = 'search?searchTerm=${props.searchTerm}'` and leave `selectedTab` and `pageIndex` as empty. Then the `SearchPage` could use a default value for these.

Reply:
the only issue with this is that the back button then requires two clicks to go back as the selectedTab and pageIndex are injected into the url. Will address this in a follow up PR.

Reply:
^ not sure I understand this, but can we chat about it on slack or in person